src/cfclient/ui/dialogs/lighthouse_bs_geometry_dialog.py

```diff
@@ -136,14 +138,24 @@ def __init__(self, lighthouse_tab, *args):
         self._lighthouse_tab = lighthouse_tab
 
         self._estimate_geometry_button.clicked.connect(self._estimate_geometry_button_clicked)
+        self._estimate_geometry_old_button.clicked.connect(self._estimate_geometry_old_button_clicked)
         self._write_to_cf_button.clicked.connect(self._write_to_cf_button_clicked)
 
         self._sweep_angles_received_and_averaged_signal.connect(self._sweep_angles_received_and_averaged_cb)
+        self._basestation_geometery_received_signal.connect(self._basestation_geometry_received_signal_cb)
         self._close_button.clicked.connect(self.close)
 
+        try:
+            import cv2
```

Comment:
This is causing flake8 to give errors in the CI. Perhaps there is a better way to check if modules exist in the environment?

Reply:
Or perhaps we should handle this completely from the cflib,